sdk/storage/Azure.Storage.Files.DataLake/src/Models/AccessControlType.cs

```diff
@@ -0,0 +1,35 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Azure.Storage.Files.DataLake.Models
+{
+    /// <summary>
+    /// Specifies the type of the <see cref="PathAccessControlEntry"/>.
+    /// </summary>
+    public enum AccessControlType
```

Comment:
We might need to assign these values, or else the default will be set to User

Reply:
fixed.